features/payment-handler.yml

```
@@ -0,0 +1,34 @@
+name: Payment Handler
```

`name` field is sentence-case
```suggestion
name: Payment handler
```